TKeazirian.HTTPServer/Handler/SimpleOptionsHandler.cs

```diff
@@ -0,0 +1,46 @@
+namespace TKeazirian.HTTPServer.Handler;
+
+using Response;
+using Request;
+
+public class SimpleOptionsHandler : Handler
+{
+    public override List<string> AllowedHttpMethods()
+    {
+        return new List<string>() { "GET" };
+    }
+
+    public override Response HandleResponse(Request request)
+    {
+        if (request.GetRequestPath() == "/method_options")
+        {
+            return new ResponseBuilder()
+                .SetStatusCode(HttpStatusCode.Ok)
+                .SetHeaders("Allow", AddHeadToAllowedMethods())
+                .Build();
+        }
+
+        return new ResponseBuilder()
+            .SetStatusCode(HttpStatusCode.Ok)
+            .SetHeaders("Allow", AddHeadPutPostToAllowedMethods())
+            .Build();
+    }
+
+    private string AddHeadToAllowedMethods()
+    {
+        List<string> allowHead = new List<string>() { "HEAD, OPTIONS" };
+        allowHead.AddRange(AllowedHttpMethods());
+
+        string allowedMethods = string.Join(", ", allowHead);
+        return allowedMethods;
+    }
+
+    private string AddHeadPutPostToAllowedMethods()
+    {
+        List<string> allowHead = new List<string>() { "HEAD", "OPTIONS", "PUT", "POST" };
+        allowHead.AddRange(AllowedHttpMethods());
+
+        string allowedMethods = string.Join(", ", allowHead);
+        return allowedMethods;
+    }
```

Comment:
I would not expect to see anything about HEAD or OPTIONS in this Handler at all.